### PR TITLE
Integrated Simultaneous Equation solver in CLI 

### PR DESCRIPTION
**Pull request by mayankDhiman**
**_Changelog:_**
+ integrated simultaneous equation solver in CLI.
+ made simultaneous equation accept arguments as tokens, as like in other modules.
+ made simultaneous equation solver solve for other variables except `x`, `y` and `z`.
+ added test cases showing above behaviour. 
+ moved the module which generates the final result string in CLI to `io/parser` with other i/o modules.

**_Test cases:_**
[Here](https://pastebin.com/QnYs8Qpe) are the test cases showing the result produced by simultaneous equation solver in CLI.

_some errors regarding displaying results in CLI, might be there, which I will be solved with next patches_

### CODE REVIEW

File: visma/solvers/simulEqn.py [hunk 1]
```diff
@@ -27,20 +28,34 @@ def coeffCalculator(LandR_tokens):
...
     return coefficients
 
 
-def getResult(matD, matDx, matDy, matDz):
-    '''Calculates values of x, y and z'''
+def getResult(matD, matDx, matDy, matDz, variables, comments, animation, solveFor):
+    '''Calculates values of x, y and z
+
+    Arguments:
+    matD {visma.matrix.structure.Matrix.SquareMat} -- Matrix Token
```
Comment (8hantanu) [inline visma/solvers/simulEqn.py]:
```suggestion
        matD {visma.matrix.structure.Matrix.SquareMat} -- Matrix Token
```
Same in `simulSolver` docstring.

File: visma/io/parser.py [hunk 2]
```diff
@@ -41,6 +41,53 @@ def resultLatex(operation, equations, comments, wrtVar=None):
...
+
+    commentsString = []
+    for x in comments:
+        if not x:
+            commentsString.append([])
+        else:
+            for y in x:
+                commentsString.append([y.translate({ord(c): None for c in '${\}'})])
+
+    finalSteps = ''
```
Comment (8hantanu) [inline visma/io/parser.py:70]: :+1: for adding step comments

**Review by 8hantanu — CHANGES_REQUESTED** [review]
You can add a default 'solve for all variable' option when the variable is not specified in your next patch.

`solve(1000x + 2y + 3z = 4; 5x + 6y + 7z = 8; 9x + 10y + 11z = 12)` must solve for x, y and z.

File: visma/io/parser.py (hunk 2 above)
Comment (mayankDhiman) [reply visma/io/parser.py:70]: Thanks :)

**Review by 8hantanu — APPROVED** [review]

**Pull request merged**